openmc/mgxs/library.py

```diff
@@ -273,6 +276,12 @@ def domains(self, domains):
             elif self.domain_type == 'universe':
                 cv.check_iterable_type('domain', domains, openmc.Universe)
                 all_domains = self.openmc_geometry.get_all_universes()
+            elif self.domain_type == 'mesh':
```

Comment:
This is cool that you can compute MGXS on multiple meshes with the same library.